cypress/e2e/supervision/navigation.cy.js

```diff
@@ -1,11 +1,10 @@
 describe("Navigation", { tags: ["@supervision", "@smoke-journey"] }, () => {
-  beforeEach(() => {
-    cy.loginAs("Case Manager");
-    cy.visit("/supervision/#/dashboard");
-  });
-
   it("log out is clicked", () => {
-    cy.get('.log-out.button').click()
-    cy.url().should('include', '/auth/login?loggedout=1')
+    cy.loginAs("Case Manager");
+    cy.visit("/supervision/#/dashboard").then(() => {
+      cy.wait(2000)
```

Comment:
`cy.get` should have an implicit wait built in, so worth checking if you really need the wait 2000 here.

Reply:
True. Without the wait it was redirecting to `http://frontend-proxy/?redirect=%2Fsupervision%2F%23%2Fdashboard`, which usually happens when Angular gets a 401 from one of the API calls. I wonder if pressing the logout button while Angular is still making calls to the API will result in one of those calls returning a 401 before the logout page gets to load in browser. So in this case I thought I'd wait for the page to fully load

Reply:
I suppose the Angular app might not be trapping and cancelling anything on page unload which could account for that weird behaviour.